lightwood/api/json_ai.py

```diff
@@ -119,7 +119,7 @@ def lookup_encoder(
             if problem_defintion.unbias_target:
                 encoder_dict["args"][
                     "target_weights"
-                ] = "$statistical_analysis.target_class_distribution"
+                ] = "$statistical_analysis.suggested_target_class_weight"
```

Comment:
just call it target_class_weights versus target_class_distributions. Target_class_weights will always refer to what is passed to the model henceforth.